### ISSUE TITLE
Allow for nested lists

### ISSUE THREAD

**Issue by 0xdf223** [rvben/rumdl#33]
There are times I want to show sublists:

1. List item 1
   - sub list 1.1
   - sub list 1.2
   - sub list 1.3
1. List item 2 
   - sub list 2.1
   - sub list 2.2
1. List item 3
   - sub 3.1
   - sub 3.2

My markdown for this triggers MD032:

```md
# TITLE

## Heading

1. List item 1
   - sub list 1.1
   - sub list 1.2
   - sub list 1.3
1. List item 2 
   - sub list 2.1
   - sub list 2.2
1. List item 3
   - sub 3.1
   - sub 3.2

That was a nice list.
```

Output:

```
$ rumdl check test7.md 
test7.md:5:1: [MD032] List should be followed by blank line [*]
test7.md:6:1: [MD032] List should be preceded by blank line [*]
test7.md:8:1: [MD032] List should be followed by blank line [*]
test7.md:9:1: [MD032] List should be preceded by blank line [*]
test7.md:9:1: [MD032] List should be followed by blank line [*]
test7.md:10:1: [MD032] List should be preceded by blank line [*]
test7.md:11:1: [MD032] List should be followed by blank line [*]
test7.md:12:1: [MD032] List should be preceded by blank line [*]
test7.md:12:1: [MD032] List should be followed by blank line [*]
test7.md:13:1: [MD032] List should be preceded by blank line [*]

Issues: Found 10 issues in 1 file (16ms)
Run with `--fix` to automatically fix 10 of the 10 issues
```

Running with `--fix` makes:

```md
# TITLE

## Heading

1. List item 1

   - sub list 1.1
   - sub list 1.2
   - sub list 1.3

1. List item 2

   - sub list 2.1
   - sub list 2.2

1. List item 3

   - sub 3.1
   - sub 3.2

That was a nice list.
```

That looks like:

1. List item 1

   - sub list 1.1
   - sub list 1.2
   - sub list 1.3

1. List item 2

   - sub list 2.1
   - sub list 2.2

1. List item 3

   - sub 3.1
   - sub 3.2

can there at least an option to allow it without the extra spacing?

**Comment by rvben** [comment]
This clearly was a bug. This should now be fixed.

**Comment by 0xdf223** [comment]
In 0.0.109?

**Comment by rvben** [comment]
You are fast! It's part of the 0.0.110 release, which was triggered just after I sent the comment. I'll try to be clearer in my communication about this next time. :)

Thanks again for raising these issues, it really helps me a lot.
